.eslintignore

```diff
@@ -1 +1,22 @@
+# NOTE: This file and .eslintignore must contain same paths.
+
+/.cache
+/art
+/coverage
+/doc
+/node/lib
 /node/src/fbs
+/rust
+/target
+/worker/deps
+/worker/fbs
+/worker/fuzzer
+/worker/include
+/worker/prebuild
+/worker/pip_invoke
+/worker/src
+/worker/subprojects
+/worker/test
+/worker/out
```

Comment:
I think I asked the same in a previous PR and I may have missed the reponse. Why not add just the `/worker` folder?

Reply:
You were replied in the previous PR. We have to lint and format JS scripts in worker folder such as worker/scripts/clang-format.mjs.

Reply:
Why can't we just include that specific file? Because Prettier suck at that. Read the detailed comments in previous Prettier PR.

Reply:
Did you report this upstream?

Reply:
No, because:

- As I said in previous PR, Prettier folks ignore reports about prettierignore stuff and those issues are auto closed due to inactivity systematically.
- .prettierignore is supposed to follow .gitignore syntax but I don't know the later in depth, meaning that I'm not sure how to ignore an entire worker folder and still un-ignore a specific worker/scripts/xxx.mjs file.
- And honestly I don't think they are gonna care.

Reply:
But ok, I'll create a test git project to verify it and be able to report wrong behavior of prettierignore.